Firestore/Example/Benchmarks/FSTLevelDBBenchmarkTests.mm

```diff
@@ -133,7 +127,7 @@ void WriteIndex(LevelDbTransaction *txn, const DocumentKey &docKey) {
   int64_t documentSize = state.range(1);
   int64_t docsToUpdate = state.range(2);
   std::string documentUpdate = UpdatedDocumentData(documentSize);
-  for (const auto &_ : state) {
+  for (auto _ : state) {
```

Comment:
Just curious, what are elements of `state`?

Reply:
I'm not aware of what the contents are. Before Google Benchmark was updated for C++11, this would have been:
```c++
while (state.KeepRunning()) {
  // ...
}
```
See the Google Benchmark's [own explanation for why one is preferred over the other](https://github.com/google/benchmark#a-faster-keep-running-loop).